LeaderboardBackend.Test/Categories.cs

```diff
@@ -68,9 +71,12 @@ public static async Task CreateCategory_GetCategory_OK()
             {
                 Body = new CreateCategoryRequest()
                 {
-                    Name = Generators.GenerateRandomString(),
-                    Slug = Generators.GenerateRandomString(),
-                    LeaderboardId = createdLeaderboard.Id
+                    Name = "1 Player",
+                    Slug = "1_player",
```

Comment:
Some additional test cases referenced here: https://github.com/leaderboardsgg/leaderboard-backend/issues/219

Better to do as isolated unit of work.